README.md

```diff
@@ -1,180 +1,75 @@
+refer to the Plugin's [documentation](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/) to get started with 
+[building](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-a-specific-target), [previewing](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/previewing-documentation),
+and publishing your documentation to your [website](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-hosting-online) or [GitHub Pages](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/publishing-to-github-pages).
 
-        ```bash
-        DOCC_EXEC=/path/to/docc swift package generate-documentation
-        ```
+**3. For documenting apps, frameworks, and packages using Xcode:**
 
-## Using `docc` to build and preview documentation
+If you want to generate an API reference for your project you can use DocC via the Xcode GUI.
+Please refer to the Xcode [documentation](https://developer.apple.com/documentation/xcode/documenting-apps-frameworks-and-packages)
+to learn the essentials of how to get started.
 
-The preferred way of building documentation for your Swift package is by using
-the Swift-DocC Plugin, or if you're using Xcode, using the "Build Documentation" command. 
+## Writing and Publishing Documentation with Swift-DocC
 
-Refer to instructions in the plugin's 
-[documentation](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/)
-to get started with [building](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-a-specific-target), [previewing](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/previewing-documentation),
-and publishing your documentation to your [website](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/generating-documentation-for-hosting-online) or [GitHub Pages](https://apple.github.io/swift-docc-plugin/documentation/swiftdoccplugin/publishing-to-github-pages).
+If you want to learn how to write and format your documentation please refer to
+[Formatting Your Documentation Content](https://www.swift.org/documentation/docc/formatting-your-documentation-content).
+For publishing go to [Distributing Documentation to Other Developers](https://www.swift.org/documentation/docc/distributing-documentation-to-other-developers).
 
-Alternatively, you can manually generate symbol graph files and invoke `docc` directly. 
-Refer to instructions in [CONTRIBUTING.md](/CONTRIBUTING.md#assembling-symbol-graphs-and-building-with-docc-directly).
-  
 ## Versioning
 
-Swift-DocC's CLI tool (`docc`) will be integrated into the Swift toolchain 
+Swift-DocC's CLI tool (`docc`) is integrated into the Swift toolchain 
 and follows the Swift compiler's versioning scheme.
 
 The `SwiftDocC` library is versioned separately from `docc`. `SwiftDocC` is under
```

Comment:
`that will help us track down the bug faster.`
->
`it will help us track down the bug faster.`